### PR TITLE
Install browsers dynamically&add exprmntl channels 

### PR DESCRIPTION
**Pull request by jugglinmike**
This is intended to resolve https://github.com/w3c/wptdashboard/issues/388 & https://github.com/w3c/wptdashboard/issues/521

### CODE REVIEW

File: src/scripts/get-browser-url.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        subprocess.check_call([
+            'gsutil', 'cp', metadata_file, 'gs://%s.json' % uri
+        ])
+    finally:
+        os.remove(artifact_file)
+        os.remove(metadata_file)
+
+
+parser = argparse.ArgumentParser(description=main.__doc__)
+parser.add_argument('--product',
```
Comment (rwaldron) [inline src/scripts/get-browser-url.py]: `--product` => `--browser-name`

File: test/upload_wpt_results_test.py [hunk 2]
```diff
@@ -358,6 +367,7 @@ def test_duplicated_results(self):
             duplicated_results['1_of_2.json']['results'][0]
         )
         returncode, stdout, stderr = self.upload('firefox',
+                                                 'stable',
                                                  '1.0.1',
                                                  'linux',
                                                  '4.0',
```
Comment (rwaldron) [inline test/upload_wpt_results_test.py]: Needs tests for `'experimental'`

File: src/scripts/upload-wpt-results.py [hunk 3]
```diff
@@ -239,6 +250,7 @@ def notify(url, secret, payload):
 parser.add_argument('--raw-results-directory', required=True)
 parser.add_argument('--platform-id', required=True)
 parser.add_argument('--browser-name', required=True)
+parser.add_argument('--browser-channel', required=True)
```
Comment (rwaldron) [inline src/scripts/upload-wpt-results.py]: Can you implement `choices: (...)` here as well? If so, then I recommend 👍

File: src/scripts/get-browser-url.py (hunk 1 above)
Comment (jugglinmike) [reply src/scripts/get-browser-url.py]: Definitely

File: src/scripts/upload-wpt-results.py (hunk 3 above)
Comment (jugglinmike) [reply src/scripts/upload-wpt-results.py]: You bet

File: test/upload_wpt_results_test.py (hunk 2 above)
Comment (jugglinmike) [reply test/upload_wpt_results_test.py]: Good catch :)

**Comment by jugglinmike** [comment]
Squashed an merged to `standalone-runner` via the following commits:

- 2de45a0 Parameterize total chunks based on speed of build
- c8952d3 Install browsers dynamically&add exprmntl channels

Thanks, @rwaldron!

**Pull request closed without merging**